packages/remix-netlify-edge/server.ts

```diff
@@ -0,0 +1,69 @@
+import { createRequestHandler as createRemixRequestHandler } from "@remix-run/deno";
+import type { ServerBuild } from "@remix-run/deno";
+import type { Context } from "@netlify/edge-functions";
+
+// Deno will exist in Deno land and we only need the minimal types for ProcessEnv
+// The types for this for Deno are defined in globals.d.ts in the Netlify template
+// @ts-ignore
+globalThis.process ||= { env: Deno.env.toObject() };
```

Comment:
What about putting this in a `globals.ts` & importing that in `index.ts` instead, like we do with our other adapters?

What's currently in `globals.d.ts` in the edge template can go there as well.